### DISCUSSION TITLE
[Documentation] where is the rules required,min,emailMatch define?

### DISCUSSION THREAD

**Discussion by hezhongfeng** [vuetifyjs/vuetify#18627] [Q&A]
### Environment

**Browsers:** Chrome 119.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
https://github.com/vuetifyjs/vuetify/blob/master/packages/docs/src/examples/v-text-field/misc-password.vue

### Expected Behavior
required,min,emailMatch is work,but I dont know where define.

### Actual Behavior
I find `max` also work, where is these rules define? 

### Reproduction Link
[https://vuetifyjs.com/en/components/text-fields/#password-input](https://vuetifyjs.com/en/components/text-fields/#password-input)

**Comment by KaelWD** [comment]
https://github.com/vuetifyjs/vuetify/blob/master/packages/docs/src/examples/v-text-field/misc-password.vue#L45

The issue list of this repo is **exclusively** for bug reports and feature requests. For general questions, please join [the Discord chat room](https://community.vuetifyjs.com), [Stackoverflow](https://stackoverflow.com/questions/tagged/vuetify.js), or [GitHub discussions](https://github.com/vuetifyjs/vuetify/discussions).
